### PR TITLE
Make ERT PDAs and give them to the right presets.

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

This PR gives the ERT the correct IDs and Job titles.

289232929-29a23b41-961f-415b-9983-020026d76860

## Why / Balance

I just didn't like that the CBURN title was "Central Commander" so I did something about it.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no cl no fun

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml [hunk 1]
```diff
@@ -460,12 +460,84 @@
...
+  name: ERT Chaplain ID card
+  components:
+  - type: Sprite
+    layers:
+    - state: gold
+    - state: ert_chaplain # we have the sprite for the id but dont have chaplain ERT equipment for now.
+  - type: IdCard
+    jobTitle: ERT Soul Officer
+  - type: Item
+    heldPrefix: gold
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml:486]: `  - type: Item`
`	 heldPrefix: gold`
    is inherited from the CentcomIDCard so these lines aren't needed iirc

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml:486]: Unfortunately not.
image
They were set on the Leader ID.
I suppose I could make a base item prototype, but that seems silly.

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml:486]: hmm, maybe I can parent the leader ID anyway and just fill the jobTitle field?

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml]: The leader is gold actually, so its sorta funky, but I parented off the chaplain ID.

**Comment by Emisse** [comment]
this doesnt need a cl

**Pull request merged**